docs/gpu.md

```diff
@@ -0,0 +1,61 @@
+# Using GPUs inside containers
+
+nerdctl provides docker-compatible NVIDIA GPU support.
+
+## Prerequisites
+
+- NVIDIA Drivers
+  - Same requirement as when you use GPUs on Docker. For details, please refer to [the doc by NVIDIA](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#pre-requisites).
+- `nvidia-container-cli`
+  - containerd relies on this CLI for setting up GPUs inside container. You can install this via [`libnvidia-container` package](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/arch-overview.html#libnvidia-container).
```

Comment:
Did you try rootless (on cgroup v1)?

I guess it needs setting `no-cgroups = true`
https://github.com/moby/moby/issues/38729#issuecomment-463493866

Reply:
It doesn't work as of now.

We might need to patch `github.com/containerd/containerd/contrib/nvidia` for allowing to pass `--no-cgroup` option to `nvidia-container-cli`. 

Containerd doesn't use `nvidia-container-runtime` (instead, it executes `nvidia-container-cli` directly) so we cannot use `/etc/nvidia-container-runtime/config.toml` for nerdctl.

Reply:
A *very hacky* workaround for this is to wrap `nvidia-container-cli` to forcefully specify `--no-cgroups`.

```
mkdir -p /opt/nvidia/bin
mv /usr/bin/nvidia-container-cli /opt/nvidia/bin/
cat <<'EOF' > /usr/bin/nvidia-container-cli
#!/bin/bash
/opt/nvidia/bin/nvidia-container-cli ${@:1:($#-1)} --no-cgroups ${@:$#}
EOF
```

Reply:
Opened https://github.com/containerd/containerd/issues/5603 for discussion

Reply:
@AkihiroSuda https://github.com/containerd/containerd/pull/5604 is merged.
Updated this PR to use `--no-cgroup` and now it works in rootless environment as well.

`replace` directive is needed in `go.mod` to forcefully point to the latest commit of containerd.